dart/lib/src/protocol/sentry_response.dart

```diff
@@ -35,73 +25,57 @@ class SentryResponse {
 
   final Map<String, String>? _headers;
 
-  Map<String, String> get other => Map.unmodifiable(_other ?? const {});
+  /// Cookie key-value pairs as string.
+  final String? cookies;
 
-  final Map<String, String>? _other;
-
-  SentryResponse({
-    this.url,
-    this.body,
-    this.redirected,
-    this.statusCode,
-    this.status,
-    Map<String, String>? headers,
-    Map<String, String>? other,
-  })  : _headers = headers != null ? Map.from(headers) : null,
-        _other = other != null ? Map.from(other) : null;
+  SentryResponse(
+      {this.bodySize,
+      this.statusCode,
+      Map<String, String>? headers,
+      String? cookies})
+      : _headers = headers != null ? Map.from(headers) : null,
+        // Look for a 'Set-Cookie' header (case insensitive) if not given.
+        cookies = cookies ??
+            headers?.entries
+                .firstWhereOrNull((e) => e.key.toLowerCase() == 'set-cookie')
```

Comment:
Why don't we just read directly from the headers as suggested before when creating the `SentryResponse`?
The Request relies on `cookies: sendDefaultPii ? request.headers['Cookie'] : null,`, but the response now does in the ctor and is case insensitive, I feel like the mix of both approaches just adds more complexity.

Reply:
Well http headers are case insensitive, so that's why the access is.
As for why here - to avoid code duplication (Dio & this integration)

Reply:
> http headers are case insensitive

Let's change for the request `Cookie` then as well?
Good point.

Reply:
changed